projects/pastanaga/src/lib/textfield/input.component.ts

```diff
@@ -50,6 +50,7 @@ const REPLACE_LT_GT = new RegExp(/[<>]/g);
 })
 export class InputComponent extends TextfieldCommon implements OnInit, AfterViewChecked, OnDestroy {
     @Input() type = 'text';
+    @Input() maxCharacters = 99;
```

Comment:
why?

Reply:
You're right, no sense. Changed to `number | undefined`.